BootloaderCorePkg/Tools/CfgDataTool.py

```diff
@@ -785,6 +803,7 @@ def Main():
                             help='Configuration binary file')
     SignParser.add_argument('-o', dest='cfg_out_file', type=str, help='Signed configuration output binary file name to be generated', required=True)
     SignParser.add_argument('-k', dest='cfg_pri_key', type=str, help='Private key file (PEM format) used to sign configuration data', required=True)
+    SignParser.add_argument('-hash', dest='hash_type', type=str, help='Hash Type with RSA signing', required=True)
```

Comment:
I think it is better to use auth_type instead hash_type.   So it can be potentially used to support new signing algo (non RSA) later on.

Reply:
Ok. I would change to Authtype.